### PR TITLE
pyfr: Add Python binding for libfreeradius API.

### PR DESCRIPTION
**Pull request by jpereira**
first version of our new Python "pyfr" module
exporting some API behaviors.

### CODE REVIEW

File: src/language/python/Makefile [hunk 1]
```diff
@@ -0,0 +1,35 @@
+-include ../../../Make.inc
```
Comment (alandekok) [inline src/language/python/Makefile:1]: Can this be integrated into the main FreeRADIUS build?  I.e. the boilermake framework?

Why does this have to be a separate `Makefile`?

The default should be to build everything, so long as the correct headers / libraries are on the system.

File: src/language/python/bootstrap.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```
Comment (alandekok) [inline src/language/python/bootstrap.sh:1]: I don't think this file is necessary.  It's targeted specifically to homebrew, which isn't good.  I think it should just be deleted, and the various build / test rules placed in a `make` file.

File: src/language/python/Makefile (hunk 1 above)
Comment (arr2036) [reply src/language/python/Makefile:1]: Python specific build system

Comment (jpereira) [reply src/language/python/Makefile]: I converted to _boiler.mk_ framework as can be seen in `src/language/python/all.mk`

**Comment by arr2036** [comment]
We'll keep the code to use as a reference (exists in origin as v4/pyfr), but the abstractions for this are still wrong after multiple rounds of feedback, and I don't trust that it's any way correct.

**Pull request closed without merging**